packages/code-du-travail-frontend/src/outils/common/Agreement/__tests__/SelectAgreement.test.tsx

```diff
@@ -46,6 +46,35 @@ describe("SelectAgreement", () => {
         })
       ).toBeInTheDocument();
     });
+    it("should not show skip option ° show message", () => {
```

Comment:
Je n'ai pas compris le `°` dans le message ?

```suggestion
    it("should not show skip option ° show message", () => {
```

Reply:
moi non plus :laughing: j'ai changé